src/components/landing/Footer.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="footer-box">
+    <div class="flex column">
+      <h1 class="heading--hero">Find the local business you wish to support</h1>
+      <div class="footer-box__search hero-input flex column align-center justify-center">
+        <div class="input-wrapper">
+          <input placeholder="Search for a local business..." class="input" type="text">
```

Comment:
pls add another class to input, input--alone (It will be ok when I merge mine and we weon't have to remembre about it)